tree/ntuple/v7/inc/ROOT/RFieldBase.hxx

```diff
@@ -473,9 +481,15 @@ protected:
    /// normalized type name and type alias
    /// TODO(jalopezg): this overload may eventually be removed leaving only the `RFieldBase::Create()` that takes a
    /// single type name
-   static RResult<std::unique_ptr<RFieldBase>> Create(const std::string &fieldName, const std::string &canonicalType,
-                                                      const std::string &typeAlias, bool continueOnError = false,
-                                                      bool emulateMissingTypes = false);
+   static RResult<std::unique_ptr<RFieldBase>>
+   Create(const std::string &fieldName, const std::string &canonicalType, const std::string &typeAlias,
+          const RCreateFieldOptions &options = {}, const RNTupleDescriptor *desc = nullptr,
+          DescriptorId_t fieldId = kTraitInvalidField);
```

Comment:
This looks odd to me: Is `kTraitInvalidField` really (semantically) a `DescriptorId_t`?

Reply:
No, this was an oversight: it was supposed to be kInvalidDescriptorId. Thanks for spotting it.